packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -6,6 +6,7 @@ const {
   describeViewports,
   getViewports,
   hideInputCaret,
+  setLocale,
```

Comment:
Should we be more verbose? setApplicationLocale?

Reply:
i can roll with that

Reply:
renamed https://github.com/cerner/terra-toolkit/pull/524/commits/49b4d2cc78c3ef39d0f7d1d335c1b7a55c336f5d